src/main/java/com/cburch/logisim/fpga/hdlgenerator/ToplevelHDLGeneratorFactory.java

```diff
@@ -249,16 +244,16 @@ public String GetSubDir() {
 
   @Override
   public SortedMap<String, Integer> GetWireList(AttributeSet attrs, Netlist Nets) {
-    SortedMap<String, Integer> Wires = new TreeMap<>();
-    int NrOfClockTrees = Nets.NumberOfClockTrees();
-    int NrOfInputBubbles = Nets.NumberOfInputBubbles();
-    int NrOfOutputBubbles = Nets.NumberOfOutputBubbles();
-    int NrOfInputPorts = Nets.NumberOfInputPorts();
-    int NrOfInOutPorts = Nets.NumberOfInOutPorts();
-    int NrOfOutputPorts = Nets.NumberOfOutputPorts();
+    final var Wires = new TreeMap<String, Integer>();
```

Comment:
variable name should start with lowercase. Please rename (more below)

Reply:
See my remark above